### PR TITLE
osrm-backend: adding head and profiles

### PR DESCRIPTION
**Pull request by kkdd**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/osrm-backend.rb [hunk 1]
```diff
@@ -26,6 +27,7 @@ def install
       system "make"
       system "make", "install"
     end
+    share.install "profiles"
```
Comment (dunn) [inline Formula/osrm-backend.rb]: Could you change `share` to `pkgshare`, please? Thanks!

**Comment by kkdd** [comment]
changed from "share" to "pkgshare"

File: Formula/osrm-backend.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class OsrmBackend < Formula
   homepage "http://project-osrm.org/"
   url "https://github.com/Project-OSRM/osrm-backend/archive/v5.4.3.tar.gz"
   sha256 "501b9302d4ae622f04305debacd2f59941409c6345056ebb272779ac375f874d"
+  head "https://github.com/Project-OSRM/osrm-backend.git", :using => :git
```
Comment (MikeMcQuaid) [inline Formula/osrm-backend.rb]: Pretty sure the `, :using => :git` is necessary here. Can you check?

Comment (ilovezfs) [inline Formula/osrm-backend.rb]: it's not if you have .git

Comment (MikeMcQuaid) [inline Formula/osrm-backend.rb]: Argh, I meant _un_necessary so: we agree. Can you remove it please @kkdd? Thanks!

Comment (ilovezfs) [inline Formula/osrm-backend.rb]: I suspected as much hehe

**Comment by kkdd** [comment]
`, :using => :git` removed.

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `osrm-backend: add head and profiles.`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**